beacon_node/beacon_chain/src/state_advance_timer.rs

```diff
@@ -245,10 +230,9 @@ async fn state_advance_timer<T: BeaconChainTypes>(
                     .await
                     .unwrap_or_else(|e| {
                         warn!(
-                            log,
-                            "Unable to prepare proposer with lookahead";
-                            "error" => ?e,
-                            "slot" => next_slot,
+                            error = ?e,
+                            slot = ?next_slot,
```

Comment:
I think we probably want to keep display formatting
```suggestion
                            slot = %next_slot,
```